server/sonar-web/src/main/js/apps/quality-profiles/details/ProfileInheritance.js

```diff
@@ -89,7 +95,7 @@ export default class ProfileInheritance extends React.PureComponent {
     });
   }
 
-  handleChangeParentClick = (event: Event) => {
+  handleChangeParentClick = (event /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)